swift_browser_ui_frontend/src/common/lang.js

```diff
@@ -667,13 +678,17 @@ let default_translations = {
         empty: "Ei valittuja tiedostoja",
         defaultKeysMessage: "Oletusarvoiset julkiset avaimet lisätty.",
         clearDrop: "Tyhjennä tiedostot",
-        cancel: "Peruuta lataus",
+        cancel: "Peruuta",
         table: {
           name: "Nimi",
           path: "Polku",
           size: "Koko",
           type: "Tyyppi",
         },
+        uploadedFiles:
+          "Uploaded files will be shared with " +
```

Comment:
Suggestion for translation:
> Lähetetyt tiedostot jaetaan kaikille [My research project]:n jäsenille.

Reply:
Alternatively:
> Lähetetyt tiedostot jaetaan kaikille jäsenille projektissa [My research project]
if we don't want to bother with different word ordering for translations

Reply:
Thanks 👍